<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>EmitDataForm</class>
 <widget class="QWidget" name="EmitDataForm">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>642</width>
    <height>505</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_6">
   <property name="spacing">
    <number>0</number>
   </property>
   <property name="margin">
    <number>0</number>
   </property>
   <item>
    <widget class="QStackedWidget" name="stackedWidget">
     <property name="currentIndex">
      <number>0</number>
     </property>
     <widget class="QWidget" name="mainPage">
      <layout class="QVBoxLayout" name="verticalLayout_7">
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_3">
         <item>
          <layout class="QVBoxLayout" name="verticalLayout">
           <item>
            <widget class="QLabel" name="label">
             <property name="text">
              <string>Data:</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QTreeView" name="emitDataView">
             <property name="editTriggers">
              <set>QAbstractItemView::NoEditTriggers</set>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <widget class="Line" name="line_2">
           <property name="orientation">
            <enum>Qt::Vertical</enum>
           </property>
          </widget>
         </item>
         <item>
          <layout class="QVBoxLayout" name="verticalLayout_5">
           <item>
            <widget class="QLabel" name="label_5">
             <property name="text">
              <string>Tulos:</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QComboBox" name="tilaBox"/>
           </item>
           <item>
            <layout class="QVBoxLayout" name="verticalLayout_3">
             <item>
              <widget class="QLabel" name="label_2">
               <property name="text">
                <string>Kilpailija:</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QLineEdit" name="kilpailijaEdit"/>
             </item>
            </layout>
           </item>
           <item>
            <layout class="QHBoxLayout" name="horizontalLayout_2" stretch="1,0">
             <item>
              <layout class="QVBoxLayout" name="verticalLayout_4">
               <item>
                <widget class="QLabel" name="label_4">
                 <property name="text">
                  <string>Sarja:</string>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QComboBox" name="sarjaBox"/>
               </item>
              </layout>
             </item>
             <item>
              <layout class="QVBoxLayout" name="verticalLayout_2">
               <item>
                <widget class="QLabel" name="label_3">
                 <property name="text">
                  <string>Aika:</string>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QTimeEdit" name="aikaTimeEdit">
                 <property name="alignment">
                  <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
                 </property>
                 <property name="readOnly">
                  <bool>true</bool>
                 </property>
                 <property name="buttonSymbols">
                  <enum>QAbstractSpinBox::NoButtons</enum>
                 </property>
                 <property name="displayFormat">
                  <string>HH.mm.ss</string>
                 </property>
                </widget>
               </item>
              </layout>
             </item>
            </layout>
           </item>
           <item>
            <spacer name="verticalSpacer">
             <property name="orientation">
              <enum>Qt::Vertical</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>20</width>
               <height>40</height>
              </size>
             </property>
            </spacer>
           </item>
           <item>
            <widget class="QLabel" name="koodi99Label">
             <property name="palette">
              <palette>
               <active>
                <colorrole role="WindowText">
                 <brush brushstyle="SolidPattern">
                  <color alpha="255">
                   <red>255</red>
                   <green>0</green>
                   <blue>0</blue>
                  </color>
                 </brush>
                </colorrole>
               </active>
               <inactive>
                <colorrole role="WindowText">
                 <brush brushstyle="SolidPattern">
                  <color alpha="255">
                   <red>255</red>
                   <green>0</green>
                   <blue>0</blue>
                  </color>
                 </brush>
                </colorrole>
               </inactive>
               <disabled>
                <colorrole role="WindowText">
                 <brush brushstyle="SolidPattern">
                  <color alpha="255">
                   <red>118</red>
                   <green>118</green>
                   <blue>117</blue>
                  </color>
                 </brush>
                </colorrole>
               </disabled>
              </palette>
             </property>
             <property name="font">
              <font>
               <pointsize>16</pointsize>
              </font>
             </property>
             <property name="text">
              <string/>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QWidget" name="tilaWidget" native="true">
             <property name="minimumSize">
              <size>
               <width>0</width>
               <height>40</height>
              </size>
             </property>
            </widget>
           </item>
          </layout>
         </item>
        </layout>
       </item>
       <item>
        <widget class="Line" name="line">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
        </widget>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout">
         <item>
          <spacer name="horizontalSpacer">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
         <item>
          <widget class="QPushButton" name="suljeTallentamattaButton">
           <property name="text">
            <string>Sulje Tallentamatta</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="closeButton">
           <property name="text">
            <string>&amp;Tallenna</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="kysymysPage">
      <layout class="QVBoxLayout" name="verticalLayout_8">
       <item>
        <widget class="QLabel" name="label_6">
         <property name="text">
          <string>Valitse korvattava tulos:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QTreeView" name="tulosView"/>
       </item>
       <item>
        <widget class="Line" name="line_3">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
        </widget>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_4">
         <item>
          <spacer name="horizontalSpacer_2">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
         <item>
          <widget class="QPushButton" name="korvaaButton">
           <property name="text">
            <string>&amp;Korvaa</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="uusiButton">
           <property name="text">
            <string>&amp;Uusi</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
